docs/ios-setup.md

```diff
@@ -0,0 +1,98 @@
+# Installation
+
+## With CocoaPods
+
+If you don't have a Podfile in the `ios` folder of your React Native project, follow this guide to create one: https://facebook.github.io/react-native/docs/integration-with-existing-apps#configuring-cocoapods-dependencies
+
+Add the following to your Podfile:
+
+  ```ruby
+  pod "react-native-heap", path: "../node_modules/@heap/react-native-heap"
+  ```
+
+Then run:
+
+  ```bash
+  pod install
+  ```
+
+You're done! :tada:
+
+## Without Cocoapods
+
+All terminal commands assume you are in the top-level directory of your React Native project.
+
+1. Locate the Xcode project for RNHeap: `open node_modules/@heap/react-native-heap/ios` . You should see a Finder window containing the `RNHeap.xcodeproj` file.
+1. Open the Xcode project for your native app, in the `ios` directory of your project.
+1. Drag the `RNHeap.xcodeproj` file from the Finder window into the "Libraries" group of your Xcode project.
+1. Select your project name in the Xcode project navigator. Select the "Build Phases" tab for your app's target.
+1. Expand the build phase called "Link Binary With Libraries". In the project navigator on the left, expand the `RNHeap.xcodeproj` entry, and find `libRNHeap.a` in the "Products" group. The icon for `libRNHeap.a` should look like a little building with columns.
+1. Drag the `libRNHeap.a` file into the list of other libraries to be linked.
+1. Expand the build phase called "Copy Bundle Resources". In the project navigator on the left, expand the `RNHeap.xcodeproj` entry, and find `HeapSettings.bundle`. Drag this into the list of bundle resources to be copied.
+
+**NOTE**: Using `react-native link` is not currently supported
+
+# Initialization
```

Comment:
Nit: maybe we should call this "configuration", not "initialization". Under the hood it's the latter but from the user's point of view it's regular configuration. Wdyt?

Reply:
Works for me 👍